force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -44,7 +52,8 @@ public with sharing class VariablesDataTypesOperators {
      * @return The difference between the two numbers.
      */
     public static Integer subtraction(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        Integer cMinus = a -b; 
```

Comment:
Add a space on either side of your operator for readability: `Integer cMinus = a - b; `